src/phoenix/server/api/input_types/SpanSort.py

```diff
@@ -29,26 +31,54 @@ class SpanColumn(Enum):
     cumulativeTokenCountPrompt = auto()
     cumulativeTokenCountCompletion = auto()
 
+    @property
+    def orm_expression(self) -> Any:
+        if self is SpanColumn.startTime:
+            return models.Span.start_time
+        if self is SpanColumn.endTime:
+            return models.Span.end_time
+        if self is SpanColumn.latencyMs:
+            return models.Span.latency_ms
+        if self is SpanColumn.tokenCountTotal:
+            return models.Span.attributes[LLM_TOKEN_COUNT_TOTAL].as_float()
+        if self is SpanColumn.tokenCountPrompt:
+            return models.Span.attributes[LLM_TOKEN_COUNT_PROMPT].as_float()
+        if self is SpanColumn.tokenCountCompletion:
+            return models.Span.attributes[LLM_TOKEN_COUNT_COMPLETION].as_float()
```

Comment:
@RogerHYang What's the reason for casting these attributes to float?

Reply:
otherwise it remains as a JSON object an can't be summed